tests/data/placebo/test_rds_delete_copy_restore/rds.ListTagsForResource_1.json

```diff
@@ -1,52 +1,52 @@
 {
-    "status_code": 200, 
+    "status_code": 200,
```

Comment:
this file shouldn't be needed anymore. does the test pass if you remove?

Reply:
No Kapil the test fails if I remove this file.  We are querying `rds-snapshot` resource in this test which is still using `list_tags_for_resource`:
```
snaps = p.resource_manager.get_resource_manager(
            'rds-snapshot').get_resources(('final-mxtest-2017-05-25',))
```

Reply:
so we would need one response for listing the tags for this test. I will delete the other one.

Reply:
that's fine then if its a correlated resource. note that will likely mean re-recording this when doing snapshots.